client/components/Supplier/Supplier.scss

```diff
@@ -1,3 +1,122 @@
 .supplier-title {
   color: blue;
+}
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: montserrat, sans-serif;
+}
+input, button {
+  appearance: none;
+  background: none;
+  border: none;
+  outline: none;
+}
+
+.supplier-title {
+  height: 100vh;
+  display: flex;
```

Comment:
Since your `supplier-title` only has one child, I don't think this will do much?